lms/djangoapps/cohort_management_dashboard/tab.py

```diff
@@ -1,4 +1,4 @@
-from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_noop as _
```

Comment:
@Golub-Sergey Please explain why do we need this change?

Reply:
for most of tabs uses `ugettext_noop`, if we use this method cohort dashboard tab translation changes correctly

Reply:
when we use `ugettex` method it translate text of tab once and then doesn't translate, if we use `ugettext_noop` method it translates always when we call it. `ugettext_noop` uses in all tabs

Reply:
Thanks for the explanation!